app/src/main/java/at/roteskreuz/stopcorona/model/db/DefaultDatabase.kt

```diff
@@ -394,6 +400,33 @@ abstract class DefaultDatabase : RoomDatabase() {
             migration(22, 23) {
                 // delete old table
                 execSQL("DROP TABLE `received_infection_message`")
+            },
+            /**
+             * Add _rollingPeriod and update primary keys in sent_temporary_exposure_keys
+             */
+            migration(23, 24) {
+                // create new temp table with added _rollingPeriod and updated primary keys
+                execSQL(
+                    """CREATE TABLE IF NOT EXISTS `sent_temporary_exposure_keys_temp` (
+                      | `rollingStartIntervalNumber` INTEGER NOT NULL, 
+                      | `_rollingPeriod` INTEGER NOT NULL, 
+                      | `password` TEXT NOT NULL, 
+                      | `messageType` TEXT NOT NULL, 
+                      | PRIMARY KEY(`rollingStartIntervalNumber`, `_rollingPeriod`)
+                      |)
+                    """.trimMargin()
```

Comment:
Seems like we are losing the IDE highlighting when using pipes and `trimMargin()`. Do we need them? Otherwise I would use the raw query without pipes.